src/schema/schema_create.c

```diff
@@ -136,12 +189,20 @@ __create_file(WT_SESSION_IMPL *session, const char *uri, bool exclusive, const c
                   uri);
             }
         }
-    } else
+    } else {
         /* Create the file. */
         WT_ERR(__wt_block_manager_create(session, filename, allocsize));
 
-    if (WT_META_TRACKING(session))
-        WT_ERR(__wt_meta_track_fileop(session, NULL, uri));
+        /*
+         * Track the creation of this file.
+         *
+         * If something down the line fails, we're going to need to roll this back. Specifically do
```

Comment:
This part is not specific to this change but it's something I noticed when running my new test.

If the import failed after this point, we were previously wiping the data file off the disk. I've made this change to only do this when we're not importing.